src/agent/mdns_mojo.cpp

```diff
@@ -125,14 +127,17 @@ void MdnsMojoPublisher::mMojoDisconnectedCb(void)
 
 otbrError MdnsMojoPublisher::Start(void)
 {
-    otbrError err = OTBR_ERROR_NONE;
-
-    VerifyOrExit(mConnector != nullptr, err = OTBR_ERROR_MDNS);
+    if (mConnector != nullptr)
+    {
+        mStateHandler(mContext, kStateReady);
```

Comment:
move this to MdnsMojoPublisher::mMojoConnectCb()?

Reply:
BorderAgent will call Start() itself when mojo is connected so we have to call handler here.